consensus/src/consensus.rs

```diff
@@ -243,6 +247,9 @@ impl<S: Storage> Consensus<S> {
         // 2. Insert/canonize block
         self.ledger.insert_and_commit(block)?;
 
+        // Increment the current block height metric
+        metrics::increment_gauge!(BLOCK_HEIGHT, 1.0);
```

Comment:
can we just always report the absolute value to avoid divergence?

Reply:
in other words, we'd just always set the gauge to the value of the atomic? I didn't see any divergence, but yeah, I don't see why not, just to be extra safe.

Reply:
the same would apply to the `queues::OUTBOUND` value; since this is spread between 3 PRs building on top of one another, I'd prefer to do it as a follow-up to avoid conflicting with myself

Reply:
yeah, it's just future proofing so we don't get the inevitable bug of some PR having it diverge as a side effect. thanks.

Reply:
I'm trying to fix this in https://github.com/AleoHQ/snarkOS/pull/965/files